app/presenters/tree_builder_tags.rb

```diff
@@ -44,17 +44,26 @@ def root_options
   end
 
   def x_get_tree_roots(count_only, _options)
+    return @categories.size if count_only
```

Comment:
That was probably an oversight. May have to put in a PR to get this line into master